plugins/Bookmark.jsx

```diff
@@ -118,6 +127,14 @@ class Bookmark extends React.Component {
             location.href = url;
         }
     };
+    zoomToBookmarkExtent = (bookmarkkey) => {
+        resolveBookmark(bookmarkkey, (params) => {
+            const scale = parseFloat(params.s);
+            const zoom = MapUtils.computeZoom(this.props.mapScales, scale);
+            const center = params.c.split(/[;,]/g).map(x => parseFloat(x));
```

Comment:
Depending on `urlPositionFormat` in `config.json`, the url parameters may contain `c` and `s`, or `e` for extent. You might also want to handle the `e` case here.